frontend/src/app/HealthChecks.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import { Route } from "react-router-dom";
+
+const HealthChecks: React.FC<{}> = ({ match }: any) => (
+  <>
+    <Route path={match.url + "/ping"} render={() => <div>pong</div>} />
+    <Route
+      path={match.url + "/commit"}
+      render={() => <div>{process.env.REACT_APP_CURRENT_COMMIT}</div>}
```

Comment:
In a production setup does this get resolved to what the env variable was at build time? I'm not sure. I guess it will be clear pretty quick if not!

Reply:
Yes, build time: https://create-react-app.dev/docs/adding-custom-environment-variables#referencing-environment-variables-in-the-html